### ISSUE TITLE
tbc 2.4.3 port

### ISSUE THREAD

**Issue by Macumbafeh** [mrrosh/HealComm#2]
Hello,

Is it possible to make this addon working on TBC 2.4.3 or atleast you can help me to do it :).

I can do all the test you want :)

Thanks

**Comment by mrrosh** [comment]
It's not my addon. I've just re-uploaded it because the original author deleted the repository.

**Comment by Macumbafeh** [comment]
Oh, who is the author then? Maybe i can ask him?

**Comment by mrrosh** [comment]
Yes, it says in the readme that it's created by [Aviana](https://github.com/Aviana)
